guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml

```diff
@@ -0,0 +1,12 @@
+title:     Inproper parsing of HTTP headers
+link:      https://github.com/guzzle/psr7/releases/tag/2.1.1
+cve:       GHSA-q7rv-6hp3-vh96
```

Comment:
This is not a CVE id. Please do not list one if there isn't one. I think @nicolas-grekas previously used one with XXX placeholders which i can live with, but this is going to cause issues with scripts processing this automatically. Should probably add something to the CI script for this? cc @stof

Reply:
@naderman see #623

Reply:
Thank you. I will remember this for next time